components/base/LogoLinks.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import { GREY_2, WHITE_PRIMARY } from "../colors";
+
+// Styles
+const Container = styled.div`
+  height: 100%;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid ${GREY_2};
+  border-radius: 5px;
+  background-color: ${WHITE_PRIMARY};
+`;
+
+const StyledTabList = styled.ul`
+  height: 100%;
+  list-style-type: none;
+  padding: unset;
+  margin: unset;
+  border-radius: 5px;
+  display: flex;
+  align-items: stretch;
+  flex-wrap: nowrap;
+  /* Tab Bar will scroll on smaller screens */
+  overflow-x: auto;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+  > :nth-of-type(1) {
+    padding-left: 40px;
+  }
+  > :nth-last-of-type(1) {
+    padding-right: 40px;
+  }
+`;
+
+const StyledLink = styled.a`
+  text-decoration: none;
+  flex-grow: 1;
+  padding: 0 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+// Props
+type TabProps = {
+  label: string;
+  icon: string;
+  iconActive: string;
+  href: string;
+};
+
+type Props = {
+  tabs: TabProps[];
+};
+
+function LogoLinks({ tabs }: Props) {
+  const [activeTab, setActiveTab] = useState(null);
+  return (
+    <Container>
+      <StyledTabList>
+        {tabs.map((tab, index) => (
+          <StyledLink
+            // eslint-disable-next-line react/no-array-index-key
+            key={index}
+            href={tab.href}
+            rel="noopener noreferrer"
+            target="_blank"
+            onMouseEnter={() => setActiveTab(tab)}
+            onMouseLeave={() => setActiveTab(null)}
+            onFocus={() => setActiveTab(tab)}
+            onBlur={() => setActiveTab(null)}
```

Comment:
Could we track focused and hovered tabs separately? I.e. instead of the one `activeTab`, there would be a `focusTab` and a `hoverTab` (or something like that). The reason for this is right now, if you tab a given logo and then hover the mouse over a different logo, the `activeTab` (and colored logo state associated with it) switches, but the focus remains on a now grayed out logo

It's definitely an edge case, but it shouldn't be a hard fix, especially since you're already doing something similar in the TabBarPanel

Reply:
Would something like [this](https://github.com/Dwolla/dev-portal/pull/68/commits/0a552e952da06bd28789ed0888627053e07a6c3d) work?

Reply:
Hell yeah!! That's awesome